src/Serializer/AbstractItemNormalizer.php

```diff
@@ -249,7 +249,22 @@ public function denormalize($data, $class, $format = null, array $context = [])
             return $item;
         }
 
-        return parent::denormalize($data, $resourceClass, $format, $context);
+        $previousObject = null !== $objectToPopulate ? clone $objectToPopulate : null;
+        $object = parent::denormalize($data, $resourceClass, $format, $context);
+
+        // Revert attributes that aren't allowed to be changed after a post-denormalize check
+        foreach (array_keys($data) as $attribute) {
```

Comment:
Was it possible to modify a property of a resource without the right to do so before?
If so, isn't it a bugfix for 2.6?

Reply:
Not that I'm aware of. This is only reverting the properties that fail the new `security_post_denormalize` check. Properties that fail the `security` check are still filtered from denormalization via `AbstractItemNormalizer::getAllowedAttributes()`